respy/tests/test_replication_kw94.py

```diff
@@ -0,0 +1,49 @@
+"""Test whether respy replicates results in Keane/Wolpin(1994)"""
```

Comment:
Can you rename the test file to ``test_replication_kw_94`` which is following the current naming convention.